Adds a description of '-disable_syslog' option to
the manual page of the Erlang Port Mapper Daemon.

Not suitable for upstream.

--- a/man/man1/epmd.1	2020-02-26 06:29:18.833629216 +0000
+++ b/man/man1/epmd.1	2020-02-26 06:36:15.084070994 +0000
@@ -61,6 +61,10 @@
 Enables debug output\&. The more \fI-d\fR\& flags specified, the more debug output you will get (to a certain limit)\&. This option is most useful when the \fIepmd\fR\& daemon is not started as a daemon\&.
 .TP 2
 .B
+\fI-disable_syslog\fR\&:
+Disables logging when \fIepmd\fR\& is started as a daemon\&. Not recommended for the general use and production\&.
+.TP 2
+.B
 \fI-daemon\fR\&:
 Starts \fIepmd\fR\& detached from the controlling terminal\&. Logging ends up in syslog when available and correctly configured\&. If the \fIepmd\fR\& daemon is started at boot, this option is definitely to be used\&. It is also used when command \fIerl\fR\& automatically starts \fIepmd\fR\&\&.
 .TP 2
